src/redis_cmd.cc

```diff
@@ -4442,7 +4455,7 @@ CommandAttributes redisCommandTable[] = {
     ADD_CMD("compact", 1, "read-only", 0, 0, 0, CommandCompact),
     ADD_CMD("bgsave", 1, "read-only", 0, 0, 0, CommandBGSave),
     ADD_CMD("flushbackup", 1, "read-only", 0, 0, 0, CommandFlushBackup),
-    ADD_CMD("slaveof", 3, "read-only", 0, 0, 0, CommandSlaveOf),
+    ADD_CMD("slaveof", 3, "read-only exclusive", 0, 0, 0, CommandSlaveOf),
```

Comment:
I think `slaveof` command should have `exclusive` attribute, it is safer, @git-hulk WDYT?

Reply:
moreover, we master_host_ without mutex lock

Reply:
we had the `slaveof_mutex` to protect the master_host_ indeed

Reply:
but there is no mutex lock when accessing it

Reply:
such as info command

Reply:
oh, I misunderstand your meaning, it's better to lock that as well.